### PR TITLE
feat(auth-server): add stripe metric timings

### LINKED ISSUE #3862: Metric timings for stripe API calls
We currently have metrics on subhub timings, our direct stripe API calls should have similar metric timings so we can break-down latency issues on our requests.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-908)
┆Issue Number: FXA-908

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* We want to know metrics on how long stripe calls take.

This commit:

* Add's a response handler to stripe to send out metric timings.

Closes #3862

### CODE REVIEW

**Comment by bbangert** [comment]
It's less than ideal to have no tests for this small fragment, and I did try constructing one. Except that it results in such an artificial situation that its not really testing that it works.

I did run this locally to verify:
```
fxa-auth-server.stripe_request:193|ms
fxa-auth-server.stripe_request:165|ms
fxa-auth-server.stripe_request:95|ms
```

Stripe types also help ensure that the response property access is valid.

File: packages/fxa-auth-server/lib/payments/stripe.js [hunk 1]
```diff
@@ -119,9 +120,15 @@ class StripeHelper {
     this.stripe = new stripe(config.subscriptions.stripeApiKey, {
       apiVersion: '2019-12-03',
       maxNetworkRetries: 3,
-      typescript: undefined,
+      typescript: true,
```
Comment (vbudhram) [inline packages/fxa-auth-server/lib/payments/stripe.js]: Is this change related?

Comment (bbangert) [reply packages/fxa-auth-server/lib/payments/stripe.js]: Oops, removing.

**Comment by bbangert** [comment]
Updated the commit for the right issue.
